README.md

```diff
@@ -29,57 +29,85 @@ for Pub/Sub Lite before doing the [quickstart](#quickstart).
 ### Quickstart
 
 In this quickstart, you will learn how to send data from a Kafka topic to
-a Pub/Sub and Pub/Sub Lite topic and vice versa, from a Kafka cluster running
+a Pub/Sub or Pub/Sub Lite topic and vice versa, using a Kafka cluster running
```

Comment:
> using a Kafka cluster
using Kafka Connect running

Reply:
Kakfa cluster = brokers, which is not what we are running in standalone mode

Reply:
Good point. Thank you.